docs/releases/1.9.txt

```diff
@@ -783,6 +783,34 @@ by ``ForeignKey`` and ``GenericForeignKey`` changed from a series of
 that ``pre_save`` and ``post_save`` signals aren't sent anymore. You can use
 the ``bulk=False`` keyword argument to revert to the previous behavior.
 
+Maximum size of a request body is now limited
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The new :setting:`DATA_UPLOAD_MAX_MEMORY_SIZE` setting limits the size that a
+request body may be before a
+:exc:`~django.core.exceptions.SuspiciousOperation` is raised. File uploads do
+not count towards this limit. Since web servers typically do not do deep
+request inspection it's not possible to limit there the amount of data Django
+would have to process excluding file uploads. This amount of data is directly
+correlated to the amount of memory needed to process the request and populate
+the GET and POST dictionaries. As such, it could be used as an attack vector
+if left unchecked. Installations that are supposed to receive form posts with
+large values should review the default value of this setting.
```

Comment:
How large is large? Under what circumstances is 2.5 MB a sane default?

The new docs are headed in the correct direction, however, I think the release notes should be minimal and the bulk of the explanation should be moved to the settings page since the explanation will still be relevant after 1.9 is ancient history.